djmoney/contrib/django_rest_framework/fields.py

```diff
@@ -40,6 +47,12 @@ def validate(self, value):
             amount = value.amount if isinstance(value, Money) else value
             return super(MoneyField, self).validate(amount)
 
+        def field_from_native(self, data, files, field_name, into):
+            super(MoneyField, self).field_from_native(data, files, field_name, into)
+            currency = data.get('{}_currency'.format(field_name), None)
```

Comment:
Hello @gbarbaten !
There are no specific reasons for supporting old Python versions (as well as old Django & DRF versions) besides it's quite easy thing to do, because, historically, we have a codebase, which works on old Django versions.
I have some thoughts about dropping support for Django < 1.8 & Python 2.6. I think we could drop it after the next release - djmoney 0.11 will be the last version to support Python 2.6 & Django < 1.8.
@benjaoming what do you think about our support policy? :)